.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+## Description
+
+<!-- Provide a summary of what the PR does and why it is being submitted. -->
+
+## Related Issue
+
+<!-- If this PR is linked to any issue, provide the issue number or description here. -->
```

Comment:
Might be a good idea to suggest adding a link to the internal Jira issue, for PRs from HashiCorp employees (they should know to do this anyways, but...)